augly/image/functional.py

```diff
@@ -1786,3 +1786,64 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    metadata: Optional[List[Dict[str, Any]]] = None,
+    level: float, 
+    max_level,
+    axis: str
+) -> Image.Image:
+    
+    """
+    Skew image with respect to X-axis or Y-axis
+    
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+        
+    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
+    
+    @param maxval: Maximum value that the operation can have. This will be 
+                scaled to level/PARAMETER_MAX.
+                
+    @param output_path: the path in which the resulting image will be stored.
+        If None, the resulting PIL Image will still be returned
+
+    @param metadata: if set to be a list, metadata about the function execution
+        including its name, the source & dest width, height, etc. will be appended
+        to the inputted list. If set to None, no metadata will be appended or returned
+
+    @returns: the augmented PIL Image
+    """
+    func_kwargs = imutils.get_func_kwargs(metadata, locals())
+    image = imutils.validate_and_load_image(image)
+    if axis == 'x-axis':
+        try:
+            level = float(np.random.uniform(low=0.1, high=level)) * max_level / 10
+            w = image.size[0]
+            h = image.size[1]
+            if np.random.uniform() > 0.5:
+                level = -level
+            aug_img = image.transform((w, h),Image.AFFINE, (1, level, 0, 0, 1, 0),
+                                    resample=Image.BILINEAR)
+            imutils.get_metadata(metadata=metadata, function_name="skew", aug_img=aug_img, **func_kwargs)
+            return aug_img
+        except:
```

Comment:
Is there a particular error you're expected/you have seen that might occur? If so, you can raise an exception with a specific error message that will help the user (not just printing "Error"). If not, I think it's better to remove the try/catch & let the function fail if the user passed something in wrong.